bool blocking
---
uint8[] return # Pickled return